.github/workflows/.size-limit.yml

```diff
@@ -8,14 +8,15 @@ jobs:
 
     steps:
       - name: Checkout Repo
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
Out of interest what does this `v4` refer to?

Reply:
Version 4 of the Github actions/checkout : https://github.com/actions/checkout